scrapper.py

```diff
@@ -44,20 +79,77 @@ def get_search_urls(self):
         pass
 
 
+class Article:
+    def __init__(self, article):
+        self.article = article
```

Comment:
you have to use the Article class implemented in core_utils module. remove this stub